bigtable-client-core-parent/bigtable-client-core/src/main/java/com/google/cloud/bigtable/grpc/io/ChannelPool.java

```diff
@@ -45,11 +49,38 @@
   /** Constant <code>LOG</code> */
   protected static final Logger LOG = new Logger(ChannelPool.class);
 
+  private static final AtomicInteger ChannelIdGenerator = new AtomicInteger();
```

Comment:
Random aside - I find it surprising that there's no existing unique identifier on the channel. 

What about just using the ManagedChannel hashCode()?